src/processors/boardgameSummaries.js

```diff
@@ -106,7 +99,7 @@ async function createBoardGameSummaries (model) {
       daysPlayedIndex[n] = 0
     }
     games.forEach(item => {
-      const dayNumber = daysBetween(startDate, new Date(item.date)) + 1
+      const dayNumber = Number.parseInt(item.date.split('-')[2])
```

Comment:
Directly use the numeric day of the date to decide the day number for a specific game.